src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -73,7 +73,7 @@ public void DesignTimeBuild ([Values(false, true)] bool isRelease)
 					new BuildItem.ProjectReference (@"..\Lib1\Lib1.csproj", lib.ProjectName, lib.ProjectGuid),
 				},
 			};
-			
+			proj.SetProperty ("_AndroidUseManagedDesignTimeResourceGenerator", "False");
```

Comment:
This confuses me. The test is `DesignTimeBuild()`, so why would this *disable* the managed generator? Does this test require refactoring to be more sensible?

Reply:
@jonpryor this is testing the old DesignTime build which should still be used on Mac/Linux. So we need to make sure we disable the new stuff. Remember we have to pass in `BuildingInsideVisualStudo` for the unit tests to get round that compiler issue.